node-require/calculate.js

```diff
@@ -0,0 +1,9 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+if (process.argv[3] === 'plus') console.log('result:', add(Number(process.argv[2]), Number(process.argv[4])));
+if (process.argv[3] === 'minus') console.log('result:', subtract(Number(process.argv[2]), Number(process.argv[4])));
+if (process.argv[3] === 'times') console.log('result:', multiply(Number(process.argv[2]), Number(process.argv[4])));
+if (process.argv[3] === 'over') console.log('result:', divide(Number(process.argv[2]), Number(process.argv[4])));
```

Comment:
Since only one can be true at a time it would be better to use `else if`s here or even a `switch1`